wrappers/python/simtk/openmm/app/forcefield.py

```diff
@@ -4192,7 +4192,10 @@ def createForcePostAmoebaBondForce(self, sys, data, nonbondedMethod, nonbondedCu
                        raise ValueError(outputString)
 
                     else:
-                        force.addStretchBend(angle[0], angle[1], angle[2], bondAB, bondCB, angleDict['idealAngle']/radian, self.k1[i], self.k2[i])
+                        if type1 == types1:
```

Comment:
I think this line should be `if type1 in types1`.  `type1` is the type of the first atom.  `types1` is a list of allowed types that the first atom could have to match these parameters.